opencensus/stats/exporters/prometheus_exporter.py

```diff
@@ -201,30 +214,17 @@ def to_metric(self, desc, view):
     def collect(self):  # pragma: NO COVER
         """Collect fetches the statistics from OpenCensus
         and delivers them as Prometheus Metrics.
-        Collect is invoked everytime a prometheus.Gatherer is run
+        Collect is invoked every time a prometheus.Gatherer is run
         for example when the HTTP endpoint is invoked by Prometheus.
         """
-        for v_data in list(self.view_data):
-            signature = view_signature(self.options.namespace,
-                                       self.view_data[v_data].view)
-            desc = self.registered_views[signature]
-            metric = self.to_metric(desc,
-                                    self.view_data[v_data].view)
-            yield metric
-
-    def describe(self):
-        """ describe will be used by Prometheus Client
-        to retrieve all registered views.
-        """
-        registered = {}
-        for sign in self.registered_views:
-            registered[sign] = self.registered_views[sign]
-        for v_data in list(self.view_data):  # pragma: NO COVER
-            if not isinstance(v_data, str):
-                signature = view_signature(self.options.namespace, v_data.view)
-                desc = self.registered_views[signature]
-                metric = self.to_metric(desc,
-                                        self.view_data[v_data].view)
+        for v_name in list(self.view_name_to_data_map):
```

Comment:
I know this is a mechanical change, but since you're touching it: we can lose the `list` here.

Reply:
SGTM, and thanks for fixing this.